dist/completer.ts

```diff
@@ -145,9 +144,6 @@ export default class GoogleStackdriverCompleter {
 
   getFilterKeys(obj, prefix, keys) {
     _.forOwn(obj, (val, key) => {
-      if (key === 'labels') {
-        key = 'label';
-      }
```

Comment:
Should any of the `.js` files be in source control?

Reply:
No all of dist/* should be ignored

Reply:
grafana guide says to commit dist/ folder: http://docs.grafana.org/plugins/developing/development/#buildscript